Author: Luca Boccassi <luca.boccassi@gmail.com>
Description: Include linux/swiotlb.h if CONFIG_ARM is defined
 swiotlb.h is needed by ARM to use dma_ops and family, but nv-linux.h only
 includes it if CONFIG_ARM64 is set. Patch nv-linux.h to also include it if
 CONFIG_ARM is set.

--- a/common/inc/nv-linux.h
+++ b/common/inc/nv-linux.h
@@ -150,7 +150,7 @@
 #include <linux/dma-mapping.h>
 #endif
 
-#if defined(CONFIG_SWIOTLB) && defined(NVCPU_AARCH64)
+#if defined(CONFIG_SWIOTLB) && (defined(NVCPU_AARCH64) || defined(NVCPU_ARM))
 #include <linux/swiotlb.h>
 #endif
 
